bin/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#
+# Usage: ./bin/deploy.sh awslabs-startup-kit-templates-deploy-v1 startup
+#
+# The first argument is the bucket and the second is the aws cli profile
```

Comment:
A comment with a short description of what this does would be useful.

Reply:
Added a SIM to flush out param checking and usage.